crates/router/src/core/payments/operations/payment_capture.rs

```diff
@@ -135,6 +138,17 @@ impl<F: Send + Clone> GetTracker<F, payments::PaymentData<F>, api::PaymentsCaptu
             .await
             .transpose()?;
 
+        let capture = match payment_attempt.capture_method {
+            Some(enums::CaptureMethod::ManualMultiple) => {
+                let new_capture = Self::create_capture(state, &payment_attempt, storage_scheme)
```

Comment:
In all places where `::create_capture` used the error case is `InternalServerError` then throw `InternalServerError` from the `create_capture` itself.